### DISCUSSION TITLE
WWAN Tx from client to server is 0 kb/s with MTU=1500.

### DISCUSSION THREAD

**Discussion by WatcherOfTheSkies** [esnet/iperf#1483] [Q&A] [answered]
Hi,

I am testing with a modem (Quectel EC25) in LTE mode. When MTU on WWAN0 interface =1500, the uplink (client to server) throughput is 0kb/s. If I change the MTU of the WWAN0 interface on the modem to 1358, the transmission is 23244.17kb/s. Any ideas as to what might be causing this?

Testing with iperf 3.12

Thanks,
Peter.

**Comment by bmah888** [comment, marked as answer]
This isn't specific to iperf3, but is it possible that encapsulation between your mobile device and the base station might be consuming bytes in the frame, so that it can no longer support a 1500-byte MTU IP packet?

With MTU set to 1500, are you able to upload files from the mobile device (not using iperf3)?

Although I'd expect the TCP stack to do some kind of PMTU discovery.

**Reply by WatcherOfTheSkies** [reply]
That gave me food for thought.
I started playing around with the **/proc/sys/net/ipv4/ip_no_pmtu_disc** parameter on the Pi and anything other than 0 (default) works really well. Although I don't fully understand what is going on it appears to be a generic issue and, as you suggested not iPerf specific. 
Thanks.
